server.js

```diff
@@ -1,30 +1,131 @@
 import express from "express";
 import cors from "cors";
+import netflixData from "./data/netflix-titles.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+    const welcomePage = {
+      Hello:
+      "Hello there! Here you'll find an API with information about 1 375 Netflix movies and TV-shows!",
+      Routes: [{
+        "/content": "GET the whole array of movies and tv-shows",
+        "/content/id/'id-number": "GET a specific movie/tv-show by it's unique id-number",
+        "/content/title/'title of movie/show": "GET movies/tv-shows by title",
+        "/content/director/'director of movie/show": "GET movies/tv-shows by director",
+        "/content/type/movie": "GET whole array of movies",
+        "/content/type/show": "GET whole array of tv-shows",
+      }]
+    }
+  res.send(welcomePage)
+  });
+
+
+//Endpoint with all available content from API
+
+app.get('/content', (req, res) => {
+ 
+  res.status(200).json({
+    data: netflixData,
+    success: true,
+  });
+  
+});
+
+
+//Endpoint with movie/tv-show id-number
+
+app.get("/content/id/:id", (req, res) => {
+  const { id } = req.params;
+
+  const contentById = netflixData.find(
+    (content) => content.show_id === +id
+  );
+
+  if (contentById) {
+    res.status(200).json({
+      data: contentById,
+      success: true,
+    });
+  } else {
+    res.status(404).json({
+      data: "The movie/tv-show you're looking for is not here.. Are you sure you've entered the correct id-number?",
+      success: false,
+    })
+  }
+        
+});
+
+
+//Endpoint with title of movie/tv-show
+    
+app.get("/content/:title", (req, res) => {
+  const { title } = req.params;
+
+  const contentByTitle = netflixData.filter(
+    (content) => content.title.toLowerCase() === title.toLowerCase()
```

Comment:
This may be a good use for .includes() since when searching for a title you may only know a couple of words, or you might miss the last letter in a word and it would still return the titles that contain the path param.